target/linux/mpc85xx/files/arch/powerpc/boot/dts/t30-w.dts

```diff
@@ -0,0 +1,384 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/include/ "fsl/p1020si-pre.dtsi"
+/ {
+	model = "watchguard,t30-w";
+	compatible = "watchguard,t30-w";
+	
+	aliases {
+		serial0 = &serial0;
+		serial1 = &serial1;
+		ethernet0 = &enet0;
+		pci0 = &pci0;
+		pci1 = &pci1;
+		led-boot = &attn_orange;
+		led-failsafe = &status_red;
+		led-running = &attn_orange;
+		led-upgrade = &status_red;
+	};
+		
+	memory {
+		device_type = "memory";
+	};
+
+	lbc: localbus@ffe05000 {
+		reg = <0x0 0xffe05000 0x0 0x1000>;
+		compatible = "fsl,p1020-elbc","fsl,elbc","simple-bus";
+		interrupts = <0x13 0x02>;
+		interrupt-parent = <&mpic>;
+
+		/* NOR Flash */
+		ranges = <0x00 0x00 0x00 0xefc00000 0x100000>;
+		
+		flash@0 {
+			#address-cells = <0x01>;
+			#size-cells = <0x01>;
+			compatible = "cfi-flash";
+			reg = <0x00 0x00 0x400000>;
+			bank-width = <0x02>;
+			device-width = <0x01>;
+
+			partition@0 {
```

Comment:
I think if you put the partition@0...partition@380000 into a "partitions" subnode (a new one with the fixed-partition compatible) the "nvmem-cells" for mac-addresses and friends will start to work?

Reply:
Let me give that a try and will force push another version

Reply:
This is correct, see [this PR](https://github.com/openwrt/openwrt/pull/4792).

Hopefully it worked out?

Reply:
If there's no uboot involvement it should work. Thing is, I've seen u-boots that need to parse the tree and has "fixed" paths.
In these case, please take the time to document what's happening. i.e: include uboot errors messages and what is broken.